cy_canvas/reducers/cyElements.js

```diff
@@ -267,6 +267,11 @@ const cyElements = (state = [], action) => {
       )
     case "UNDO":
       return SessionStorage.popState()
+    case "COPY":
+      SessionStorage.saveCopiedNodes(action.elements)
+      return state
+    case "PASTE":
+      return state.concat(action.elements)
```

Comment:
I'll leave this to another card to handle all the situation that involves selected status change.